apps/docs/src/app/core/component-docs/button/examples/button-menu-example.component.html

```diff
@@ -1,14 +1,14 @@
-<button fd-button [fdMenu]="true" [glyph]="'cart'" label="Add to Cart"></button>
-<button fd-button [fdMenu]="true" [fdType]="'emphasized'" [glyph]="'cart'" label="Add to Cart"></button>
-<button fd-button [fdMenu]="true" [fdType]="'ghost'" [glyph]="'cart'" label="Add to Cart"></button>
-<button fd-button [fdMenu]="true" [fdType]="'positive'" [glyph]="'filter'" label="Filter"></button>
-<button fd-button [fdMenu]="true" [fdType]="'negative'" [glyph]="'filter'" label="Filter"></button>
-<button fd-button [fdMenu]="true" [fdType]="'attention'" [glyph]="'filter'" label="Filter"></button>
-<button fd-button [fdMenu]="true" [fdType]="'transparent'" [glyph]="'filter'" label="Filter"></button>
+<button fd-button [fdMenu]="true" glyph="cart" label="Add to Cart" ariaLabel="Add to cart"></button>
```

Comment:
those updates of the doc where `[glyph]` is replaced with `glyph` should be done in separate PR